src/main/java/net/ccbluex/liquidbounce/features/module/modules/player/Blink.kt

```diff
@@ -115,12 +156,16 @@ object Blink : Module("Blink", ModuleCategory.PLAYER, gameDetecting = false) {
     }
 
     override val tag
-        get() = packets.size.toString()
+        get() = (packets.size + packetsReceived.size).toString()
 
     private fun blink() {
+        isCleaning = 1
+        handlePackets(*packetsReceived.toTypedArray())
+        isCleaning = 0
```

Comment:
Since when are we in Python / C?

Are you sure handlePackets triggers onPacket event? I feel like isCleaning is unnecessary.

Reply:
> Since when are we in Python / C?
> 
> Are you sure handlePackets triggers onPacket event? I feel like isCleaning is unnecessary.

It does, I had an infinite loop because of that at one point